test_scripts/Polices/External_UCS/027_ATF_P_TC_LPT_Update_PreloadedPT_without_disallowed_by_external_consent_entities_on_struct.lua

```diff
@@ -0,0 +1,141 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: PreloadedPT without "disallowed_by_external_consent_entities_on" struct
+--
+-- Description:
+-- In case:
+-- SDL uploads PreloadedPolicyTable without "disallowed_by_external_consent_entities_on:
+-- [entityType: <Integer>, entityId: <Integer>]" -> of "<functional grouping>" -> from "functional_groupings" section
+-- SDL must:
+-- a. consider this PreloadedPT as valid (with the pre-conditions of all other valid PreloadedPT content)
+-- b. do not create this "disallowed_by_external_consent_entities_on: [entityType: <Integer>, entityId: <Integer>]"
+-- field of the corresponding "<functional grouping>" in the Policies database.
+--
+-- Preconditions:
```

Comment:
@dboltovskyi, I think that as precondition you should have:
1. For first ignition cycle disallowed_by_external_consent_entities_on existing in sdl_snapshot.json - Based on requirement:"[Policies] External UCS: PreloadedPT with "disallowed_by_external_consent_entities_on" struct"
2. After that to update sdl_preloaded_pt.json without this section.
3. Check listed steps.